docs/src/index.md

```diff
@@ -0,0 +1,20 @@
+# Arblib.jl Documentation
+
+This package is a wrapper around [Arb](http://arblib.org) - a C
+library for arbitrary-precision ball arithmetic. Other wrappers of Arb
+for Julia include [Nemo](https://github.com/Nemocas/Nemo.jl) and
+[ArbNumerics.jl](https://github.com/JeffreySarnoff/ArbNumerics.jl).
+
+The **goal** of Arblib.jl is to supply a **low lever wrapper** of the
+methods in Arb as well as a **high level interface**. The low level
+wrapper should allow for writing methods using mutability and with
+performance very close to that of those written in C. The high level
+interface should make it easy to use in generic Julia code, similarly
+to how `BigFloat` is a wrapper around the MPFR library. In addition it
+should be possible to seamlessly switch between the high level
+interface and the low level wrapper when needed.
```

Comment:
hehe, this **high level interface** is not something we had in mind when we started, but why not, if you have time and motivation ;)

Reply:
With high level interface I only really mean that we overload the standard Julia functions like `*` and `sin`, so that you don't have to only work directly with the low-level methods like `Arblib.mul!` and `Arblib.sin!`. And this we already do!